src/components/Dashboard/ThisWeek/PartnerCare/PartnerCare.tsx

```diff
@@ -361,19 +361,19 @@ const PartnerCare = ({
                     ))}
                   </CardList>
                   <CardActions>
-                    <Link
+                    <Button
+                      LinkComponent={NextLink}
                       href={`/accountLists/${accountListId}/tasks?filters={"activityType":["PRAYER_REQUEST"]}&completed=false`}
-                      passHref
+                      size="small"
+                      color="primary"
```

Comment:
Isn't `shallow` for when you link to the same page but with different query params? I might be misunderstanding its purpose though.

Reply:
The shallow prop updates the URL without triggering a full page reload or re-fetching data.

Reply:
Isn't `shallow` for when you link to the same page but with different query params and don't want to run `getServerSideProps` again? I might be misunderstanding its purpose though.

Reply:
Even if I add `shallow`, I still see the request for the tasks page props. It has to fetch that data because it's navigating to a new page. I think that `shallow` is for when you're linking to the same page just with different query params and don't want to fetch the props again.

Reply:
I got you; the Next.Js documentation is confusing as they don't specify that is the case. But I understand